analyzer/src/test/kotlin/managers/GoModTest.kt

```diff
@@ -25,68 +25,71 @@ import io.kotest.matchers.collections.containExactlyInAnyOrder
 import io.kotest.matchers.should
 import io.kotest.matchers.shouldBe
 
-import org.ossreviewtoolkit.model.Identifier
 import org.ossreviewtoolkit.model.VcsType
 
 class GoModTest : WordSpec({
     "toVcsInfo" should {
         "return the VCS type 'Git'" {
-            val id = Identifier("Go::github.com/chai2010/gettext-go:v1.0.0")
-
-            id.toVcsInfo().type shouldBe VcsType.GIT
+            GoMod.ModuleInfo(
```

Comment:
Mind introducing a `moduleInfo` variable here (like below) even if only a single properly is asserted? That's also how it was before with `id`.